src/main/resources/liquibase/changelog/20200110142314_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="user (generated)" id="1578662598381-1">
+                <addColumn tableName="field_config">
+            <column name="unique_field" type="BIT">
+                <constraints nullable="false"/>
```

Comment:
There are existing rows in this table if you add not null column to it, the migration will fail with the not null constraint violation, because it's null by default. One way to fix this will be to add the column without this constraint set the value for all the rows (like it's done in the next change set) and then add the not null constraint (in a separate change set).